src/main/java/net/snowflake/ingest/streaming/internal/RowBufferStats.java

```diff
@@ -330,14 +326,10 @@ byte[] performConversion(String input) {
   }
 
   void reset() {
-    this.currentMaxStrValue = null;
-    this.currentMinStrValue = null;
```

Comment:
It's a bit confusing why we only have `currentMaxColStrValue` and `currentMinColStrValue` but no `currentMaxStrValue` and `currentMinStrValue` since there is no difference in the SDK now for collated and non-collated strings, could we remove currentMaxColStrValue and currentMinColStrValue instead?

Reply:
I had to leave one of the two sets in and since in the case of collation these are the min/max according to the collation I left them that way